### PR TITLE
ASB-1454: consume nsip-documents

### PR DESCRIPTION
**Pull request by JamesP-KC**
- adds a `Document` table to Applications database, to store document data from Back Office
- adds function to subscribe to document publish events, and store the data in aforementioned table

### CODE REVIEW

File: packages/back-office-subscribers/nsip-documents/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = async (context, message) => {
+	context.log('Hello from nsip-documents function');
```
Comment (jonathangoulding) [inline packages/back-office-subscribers/nsip-documents/index.js]: log needed ?

**Review by jonathangoulding — APPROVED** [review]

File: packages/back-office-subscribers/nsip-documents/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+	context.bindings.document = {
+		...message,
+		modifiedAt: new Date()
+	};
```
[the remark this answers is not in the document]
Comment (JamesP-KC) [reply packages/back-office-subscribers/nsip-documents/index.js:5]: yeah no explicit error handling needed here as we don't want do anything special in an error case. any errors thrown in the function code are caught and logged outside of this context.

**Pull request merged**